scripts/orphan-tags.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+process.env.NODE_ENV = 'local';
+const path = require('path');
+const Mongoose = require('mongoose');
+const RestHapi = require('rest-hapi');
+const Glue = require('@hapi/glue');
+const Manifest = require('../config/manifest.conf');
+const Config = require('../config');
+const restHapiConfig = Config.get('/restHapiConfig');
+
+require('dotenv').config();
+
+(async function processTags() {
+  RestHapi.config.loglevel = 'DEBUG';
+  const Log = RestHapi.getLogger('orphan-tags.js');
+  try {
+    Mongoose.connect(restHapiConfig.mongo.URI);
+    RestHapi.config = restHapiConfig;
+    RestHapi.config.absoluteModelPath = true;
+    RestHapi.config.modelPath = path.join(__dirname, '/../server/models');
+    let models = await RestHapi.generateModels(Mongoose);
+    const composeOptions = {
+      relativeTo: path.join(__dirname, '/../'),
+    };
+    const manifest = Manifest.get('/');
+    const server = await Glue.compose(manifest, composeOptions);
+    await server.start();
```

Comment:
Just as a note, starting up a hapi server is not strictly required for this script since we are not actually calling any endpoints but just using the mongoose wrapper methods.

For more info see:
https://resthapi.com/docs/mongoose-wrapper-methods.html